src/us/kbase/workspace/test/workspace/WorkspaceLongTest.java

```diff
@@ -94,10 +94,10 @@ public void saveWithBigData() throws Exception {
 		
 		//printMem("*** released refs ***");
 		
-		ByteArrayFileCache newdata = ws.getObjects(
+		final ByteArrayFileCache newdata = ws.getObjects(
 				userfoo,
 				Arrays.asList(ObjectIdentifier.getBuilder(bigdataws).withID(1L).build()))
-				.get(0).getSerializedData();
+				.get(0).getSerializedData().get();
 //		printMem("*** retrieved object ***");
 //		System.gc();
 //		printMem("*** ran gc after retrieve ***");
```

Comment:
assuming these on your to-delete-at-some-point list

Reply:
Not really. Part of the point of that test with massive objects is to check for memory usage wrt GC, so those printouts are useful. I suppose I could make a toggle for them rather than have them as comments